_layouts/sub-intro.html

```diff
@@ -8,8 +8,21 @@
 
   <section id="main-content" class="main-content">
     <div class="callout callout-success">
-      <p>This documentation targets the API <b>shipped with Pentaho 8.2</b>. You can access the documentation regarding <a href="{{ "/8.1" | relative_url }}">Pentaho 8.1</a>, <a href="{{ "/8.0" | relative_url }}">Pentaho 8.0</a> and <a href="{{ "/7.1" | relative_url }}">Pentaho 7.1</a>.</p>
-      <p>Also check <a title="What's new and changed in Visualization API beta 5" href="{{ "/platform/whats-new-beta-4" | relative_url }}">What's new and changed in the Pentaho Platform JavaScript APIs beta 4</a>.</p>
+      <div class="callout callout-success">
```

Comment:
looking at 8.2 file, it seems this should be a <p> tag. or it is just duplicated